src/stdlib/parse_influxdb.rs

```diff
@@ -0,0 +1,332 @@
+use std::collections::BTreeMap;
+
+use chrono::DateTime;
+use influxdb_line_protocol::{FieldValue, ParsedLine};
+
+use crate::compiler::prelude::*;
+
+fn influxdb_lines_to_metrics(line: ParsedLine) -> Vec<ObjectMap> {
```

Comment:
I tried to match the Vector's `influxdb` decoder implementation
https://github.com/vectordotdev/vector/blob/master/lib/codecs/src/decoding/format/influxdb.rs